app/app/settings.py

```diff
@@ -271,7 +272,60 @@
     }
 }
 
-CACHES = {'default': env.cache()}
+CACHEOPS_DEGRADE_ON_FAILURE = env.bool('CACHEOPS_DEGRADE_ON_FAILURE', default=True)
+CACHEOPS_DEFAULTS = {
+    'timeout': 60 * 60
+}
+
+# 'all' is an alias for {'get', 'fetch', 'count', 'aggregate', 'exists'}
+CACHEOPS = {
+    '*.*': {
+        'timeout': 60 * 60,
+    },
+    'auth.user': {
+        'ops': 'get',
+        'timeout': 60 * 15,
+    },
+    'auth.group': {
+        'ops': 'get',
+        'timeout': 60 * 15,
+    },
+    'auth.*': {
+        'ops': ('fetch', 'get'),
+        'timeout': 60 * 60,
+    },
+    'auth.permission': {
+        'ops': 'all',
+        'timeout': 60 * 15,
+    },
+    'dashboard.bounty': {
```

Comment:
it makes me purdy nervous to have these objects cached; 

per https://github.com/Suor/django-cacheops it looks like the invalidation is pretty good though?